sdk/anomalydetector/ai-anomaly-detector-rest/package.json

```diff
@@ -54,8 +51,8 @@
     "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser",
     "test:node": "npm run clean && npm run build:test && npm run unit-test:node",
     "unit-test": "npm run unit-test:node && npm run unit-test:browser",
-    "unit-test:browser": "dev-tool run test:browser",
-    "unit-test:node": "dev-tool run test:node-ts-input -- --timeout 1200000 --exclude 'test/**/browser/*.spec.ts' 'test/**/*.spec.ts'",
```

Comment:
Also make sure our `hookTimeout` and `testTimeout` are set for the `vitest` configs for both browser and node.